src/full_client.toit

```diff
@@ -257,9 +257,22 @@ abstract class ReconnectionStrategyBase implements ReconnectionStrategy:
     Duration --s=5,
     Duration --s=15,
   ]
+  /**
+  The default duration to consider a new connection attempt to be independent
+    from previous attempts.
+
+  If there is a connection attempt within this duration, then the strategy
+    assumes that we managed to connect, but got disconnected immediately after.
+    It then treats this as if we didn't manage to connect at all, increasing
+    the attempt-counter and thus the timeouts.
+  */
+  static DEFAULT-RESET-DURATION_ /Duration ::= Duration --s=1
```

Comment:
The `DEFAULT-RECEIVE-CONNECT-TIMEOUT` is public. Should this be public too?

Reply:
done.